### ISSUE TITLE
It crashes directly when I turn on any setting.

### ISSUE THREAD

**Issue by SergioCupido** [vesperlol/Vespy-Logger0.3#2]
![image](https://user-images.githubusercontent.com/97339303/157288413-902b8b2e-7ab0-4992-9b5f-f48a83fbacac.png)

**Comment by vesperlol** [comment]
> ![image](https://user-images.githubusercontent.com/97339303/157288413-902b8b2e-7ab0-4992-9b5f-f48a83fbacac.png)

Im using pastebin to host my codes. It might be your country that blocks it, idk, join our server so we can help you https://discord.gg/apquteSB

**Comment by SergioCupido** [comment]
Pastebin is banned in Turkey.

**Comment by vesperlol** [comment]
> Pastebin is banned in Turkey.

Use a vpn then

**Comment by vibenOfficial** [comment]
@vesperlol i agree, there are free vpns out there (like windscribe https://windscribe.com)
